### PR TITLE
Feature/middlesex

### PR DESCRIPTION
**Pull request by kengie**
Standardized field entries for race, sex, ethnicity for linking, transitioning Middlesex flights partially to EDM v2.

### CODE REVIEW

File: MiddlesexMA/src/main/java/com/openlattice/middlesex/MiddlesexCharges2.java [hunk 1]
```diff
@@ -98,29 +102,38 @@ public static void main( String[] args ) throws InterruptedException {
...
         Flight flight = Flight.newFlight()
                 .createEntities()
 
-                .addEntity( "suspect" )
+                .addEntity( "arrestee" )
                 .to( "LPDArrestSuspects" )
                 .addProperty( new FullQualifiedName( "nc.SubjectIdentification" ) )
-                .value( MiddlesexCharges2::getSubjectIdentification ).ok()
+                .value(row -> {
+                    return Parsers.getAsString(row.getAs("Name")) + " " + Parsers.getAsString(row.getAs("DOB"));
```
Comment (katherinebernstein) [inline MiddlesexMA/src/main/java/com/openlattice/middlesex/MiddlesexCharges2.java]: is name+DOB guaranteed to be unique for each person?

Comment (kengie) [reply MiddlesexMA/src/main/java/com/openlattice/middlesex/MiddlesexCharges2.java]: I think it is the most unique combination given the limited data there is here - before the person ID was the booking number, which I confirmed in the csvs was definitely not unique - high utilizers appeared in multiple bookings, and this was making the top utilizer results erroneous. Basically, the top utilizer was always only in 1 arrest, which was wrong. Here I checked the first few top utilizers in Excel by hand, and it matched what is now in gallery.

Comment (katherinebernstein) [reply MiddlesexMA/src/main/java/com/openlattice/middlesex/MiddlesexCharges2.java]: That's the problem linking is supposed to fix -- unless Middlesex has told use that name+dob is safe to use as a unique combo, we should probably just assign each person a random id and let linking decide whether or not to merge people

Comment (kengie) [reply MiddlesexMA/src/main/java/com/openlattice/middlesex/MiddlesexCharges2.java]: I will change it - although it seems like pre-linking, it is working perfectly right now.

**Pull request merged**